code/modules/mob/mob_defines.dm

```diff
@@ -110,6 +110,7 @@
 	var/list/abilities = list()         // For species-derived or admin-given powers.
 	var/list/speak_emote = list("says") // Verbs used when speaking. Defaults to 'say' if speak_emote is null.
 	var/emote_type = 1		// Define emote default type, 1 for seen emotes, 2 for heard emotes
+	var/treadmill_speed = 1 //1 for most player things, simple animals get lower, xenos get higher
```

Comment:
I don't understand why it's not worth it. This saves many lines of code, empowers admins, and increases performance speed.